docs/development/local-development.md

```diff
@@ -46,11 +52,11 @@ The following steps work to set up a brand new Windows 10 installation for devel
 - In an Administrator PowerShell prompt, run `npm install -global npm` and then `npm --add-python-to-path='true' --debug install --global windows-build-tools`
 - Install [Yarn](https://yarnpkg.com/lang/en/docs/install/#windows-stable)
 - Install `mock` for Python:
-
   - `curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py`
   - `python get-pip.py`
   - `rm get-pip.py`
   - `python -m pip install mock`
+- Install [Java](https://adoptopenjdk.net/?variant=openjdk11)
```

Comment:
Maybe we should only specify to install latest java tls? eg adoptjdk, openjdk or any other compatible.

Reply:
I am not sure what you are suggesting. The text only says “Install Java”. I linked to one specific distribution of Java for those people that are not familiar with the discussion of different Java distributions.

If your comment is about the `?variant=openjdk11` part, I added this because the AdoptOpenJDK page pre-selects Java 8 otherwise. With the link I gave, you still get a form to select the version you want, but Java 11 is pre-selected.

Reply:
Link to adopt is ok, we should just mention that the user can also use another compatible jdk

Reply:
Still don’t get, we also give `openjdk` for Ubuntu users, even though there are other distributions available. I changed it to

```markdown
- Install Java, e.g. from [AdoptOpenJDK](https://adoptopenjdk.net/?variant=openjdk11) or any other distribution
```

Is that alright?